demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementController.php

```diff
@@ -2405,11 +2405,15 @@ public function importStatementsAction(
                 $this->getMessageBag()->add('error', 'error.missing.data',
                     ['fileName' => $fileName]);
             } catch (UnexpectedWorksheetNameException $exception) {
-                $this->getMessageBag()->add('error', 'error.worksheet.name',
-                    [
-                        'worksheetTitle' => $exception->getIncomingTitle(),
-                        'expectedTitles' => $exception->getExpectedTitles(),
-                    ]);
+                if ('Abschnitte' == $exception->getIncomingTitle()) {
```

Comment:
this can be ===

Reply:
done [3a1aa38](https://github.com/demos-europe/demosplan-core/pull/1902/commits/3a1aa38869b386757afc154f8340ab39c0b97080)